sci-libs/libqalculate/libqalculate-5.0.0b.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Bump with sci-calculators/qalculate-gtk and sci-calculators/qalculate-qt
+
+inherit toolchain-funcs
+
+MY_PV="${PV//b/}"
+
+DESCRIPTION="A modern multi-purpose calculator library"
+HOMEPAGE="https://qalculate.github.io/"
+SRC_URI="https://github.com/Qalculate/${PN}/releases/download/v${PV}/${P}.tar.gz"
+S="${WORKDIR}"/"${PN}-${MY_PV}"
+
+LICENSE="GPL-2+"
+# SONAME changes pretty often on bumps. Check!
+SLOT="0/23"
```

Comment:
Just to double check, because I'm a bit confused now:

- 5.0.0 did not bump the SONAME version number when it should have.
- this beta release, 5.0.0b, fixes this problem? Or are we simply changing the slot to workaround the problem regardless of the SONAME?

Reply:
5.0.0.b is just a new bump to bump the SO version, `qa-cmp`'ll show the SONAME differences:
```
# qa-cmp sci-libs/libqalculate-5.0.0 libqalculate-5.0.0b
...
 SONAME:-libqalculate.so.22(64)
 SONAME:+libqalculate.so.23(64)
```

Reply:
FWIW, I don't think it's a beta, just a followup release(?).